app/Lesson_word.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Lesson_word extends Model
```

Comment:
Please don't use underscore For class name, we use camel case e.g LessonWord

Reply:
Please follow Coding style standards base from this article
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md